examples/node/cli.rs

```diff
@@ -960,34 +961,41 @@ fn list_payments(node: &LightningNode, words: &mut dyn Iterator<Item = &str>) ->
         .unwrap_or("2")
         .parse()
         .context("Number of payments should be a positive integer number")?;
-    let payments = node.get_latest_payments(number_of_payments)?;
-    let pending_payments = payments.pending_payments;
-    let completed_payments = payments.completed_payments;
+    let payments = node.get_latest_movements(number_of_payments)?;
```

Comment:
is there a specific reason for still calling this variable `payments` instead of `movements`?

Reply:
Fixed in [1adb3b6](https://github.com/getlipa/lipa-lightning-lib/pull/868/commits/1adb3b6f3037e42c71db64ab8034c1632d2ecf6a)

Reply:
Method and command is still named `[...]payments` for consistency I would change this as well.

Reply:
Fixed in [299110d](https://github.com/getlipa/lipa-lightning-lib/pull/868/commits/299110d9443038ce627817d807543c60ca68efa3)